frappe/translations/fr.csv

```diff
@@ -3774,12 +3774,10 @@ Quarterly,Trimestriel,
 Queued,File d'Attente,
 Quick Entry,Écriture Rapide,
 Reason,Raison,
-Refreshing,Rafraîchissant,
+Refreshing,Actualisation,
```

Comment:
the previous one was/is correct? sorry i don't know any french but according to google it does seem to be something else entirely

<img width="783" alt="Screenshot 2022-08-25 at 11 06 06 AM" src="https://user-images.githubusercontent.com/32034600/186583216-30f68f04-c312-430d-93f2-7e1a01b63a2d.png">

Reply:
Refreshing (EN) is can be use to say "The screen is reloading" or "this drink is refreshing"

In french Rafraîchissant is for "this drink is refreshing".

the meaning into Frappe is more the "List view is reloading", in this case we use the word "Actualisation"